python/mp/effects/__init__.py

```diff
@@ -1,19 +1,23 @@
 #subpackage 'effects' for mp module (Mega Prayer)
-from mp.effects import effect
-from mp.effects import sine_wave
-from mp.effects import throb
-from mp.effects import bounce
-from mp.effects import set_color
-from mp.effects import casino
+#from mp.effects import effect
+#from mp.effects import sine_wave
+#from mp.effects import throb
+#from mp.effects import bounce
+#from mp.effects import set_color
+#from mp.effects import casino
```

Comment:
It looks like we can remove all of these imports and __init__.py still works. Its one less thing to keep in sync. I commented them out to test - maybe there's a need I don't know about?